Penteract/Assets/Scripts/PlayerController.h

```diff
@@ -155,6 +156,9 @@ class PlayerController : public Script {
 	std::vector<GameObject*> switchCollisionedGO;
 	bool switchFirstHit = true;
 
+	//Upgrades
+	GameObject* upgradeEffects = nullptr;
```

Comment:
This game object can be local to the Start method, is the only method that will be used.